test/foundry/invariant/handlers/DepositManagementHandlerTest.t.sol

```diff
@@ -1,55 +1,72 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.24;
 
-import "../base/BaseInvariantTest.t.sol";
+import { InvariantBaseTest } from "../base/InvariantBaseTest.t.sol";
+import "../../utils/Imports.sol";
 
-// DepositManagementInvariantHandler manages deposit operations for a Nexus account,
-// ensuring invariants remain intact throughout the process.
-contract DepositManagementHandler is BaseInvariantTest {
+/// @title DepositManagementHandlerTest
+/// @notice Manages deposit operations for a Nexus account, ensuring invariants remain intact throughout the process.
+contract DepositManagementHandlerTest is InvariantBaseTest {
     Nexus internal nexusAccount;
     Vm.Wallet internal signer;
 
-    // Initializes the handler with a Nexus account and wallet used for signing transactions
+    /// @notice Initializes the handler with a Nexus account and wallet used for signing transactions.
+    /// @param _nexusAccount The Nexus account to manage deposits for.
+    /// @param _signer The wallet used for signing transactions.
     constructor(Nexus _nexusAccount, Vm.Wallet memory _signer) {
         nexusAccount = _nexusAccount;
         signer = _signer;
     }
 
-    // Handles a deposit operation while verifying that the state remains consistent
+    /// @notice Handles a deposit operation while verifying that the state remains consistent.
+    /// @param amount The amount to deposit.
     function invariant_handleDeposit(uint256 amount) public {
         Execution[] memory executions = new Execution[](1);
         executions[0] = Execution({ target: address(nexusAccount), value: amount, callData: abi.encodeWithSignature("addDeposit()") });
 
         // Execute deposit through the ENTRYPOINT with invariant checks
-        PackedUserOperation[] memory userOps = preparePackedUserOperation(signer, nexusAccount, EXECTYPE_DEFAULT, executions);
+        PackedUserOperation[] memory userOps = buildPackedUserOperation(
```

Comment:
some places it's replaced with buildPackedUserOperation, whereas some place it's replaced with buildUserOperation

Reply:
`buildUserOperation` is only on Arbitrum fork testing, as it is used to create a v0.6 compatible userOp

Reply:
then we should go with build/prepare useropeartion/packeduseroperation